File: src/learn/signup.cc
#ifndef LEARN_SIGNUP_H
#define LEARN_SIGNUP_H
#include <iostream>

namespace torch
{
namespace learn
{
namespace signup
{
void signup(std::__cxx11::string& firstName,
            std::__cxx11::string& lastName,
            std::__cxx11::string& email,
            std::__cxx11::string& password,
            std::__Cxx11::string& confirmPassword);
} // namespace signup
} // namespace learn
} // namespace torch
#endif // LEARN_SIGNUP_H

File: src/learn/signup.py
class signup():
    def __init__(self, firstname, lastname, email, password, confirmPassword, hidden):
        self.firstname = firstname
        self.lastname = lastname
        self.email = email
        self.password = password
        self.confirmPassword = confirmPassword
        self.hiddenInput = hidden
    
    def formBuilder(self, input):
        input.add(self.firstname)
        input.add(self.lastname)
        input.add(self.email)
        input.add(self.password, self.hiddenInput)
        input.add(self.confirmPassword, self.hiddenInput)
        input.save('signup/output/formBuilder.app.out')